sklearn/decomposition/_online_lda_fast.pyx

```diff
@@ -44,6 +59,8 @@ def _dirichlet_expectation_1d(np.ndarray[ndim=1, dtype=np.float64_t] doc_topic,
 
     size = doc_topic.shape[0]
 
+    # Type of doc_topic and out are identical.
+    # So use the type for calculation
```

Comment:
Is this comment true? Since `doc_topic` and `out` use two different fused types, I think they can be different types.

Reply:
Thanks! It just a wrong comment. I will remove it.

Reply:
Removed.